packages/analytics-js-plugins/jest.config.mjs

```diff
@@ -3,6 +3,7 @@ export default {
   displayName: '@rudderstack/analytics-js-plugins',
   preset: '../../jest.preset.js',
   rootDir: './',
+  coveragePathIgnorePatterns: ['src/errorReporting/event'],
```

Comment:
Did you check if this overrides the value in the preset or extends it?
We'd want the latter.

Reply:
We can check in Jest's documentation. Additionally, you can test by including and excluding known source code file paths.